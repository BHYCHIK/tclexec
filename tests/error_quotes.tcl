set a 1
set b "$a
  before
  $b
  after
"
set c 2
