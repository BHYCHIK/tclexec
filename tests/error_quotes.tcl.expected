Line: 4, position: 4: expand variable b doesn't exist
  $b
   ^
